### DISCUSSION TITLE
Some problems with the bsp protocol

### DISCUSSION THREAD

**Discussion by 1234567845246** [microsoft/vscode-discussions#911] [Extension Development QnA]
For example, I have two sets of tool chains gcc and llvm, and their output information formats are different. bsp can combine the output information of them into one. Also is when can the bsp protocol development plug-in package

**Comment by isidorn** [comment]
Please provide exact reproducible steps - for example your repository. And exact steps. Thank you

**Comment by burkeholland** [comment]
Because this issue is now 3 weeks old without update, we’ll be closing it for now. If you feel this is an error and this item is stil an issue for you, please feel free to re-open.
